### ISSUE TITLE
Improve contributor docs on testing

### ISSUE THREAD

**Issue by snOm3ad** [rustwasm/wasm-bindgen#3619]
### Describe the Bug

When testing `wasm-bindgen` using `cargo test --target wasm32-unknown-unknown` the tests located in `tests/wasm/` fail with a somewhat cryptic error message:

```
Caused by:
    0: failed to generate bindings for import of `__wbindgen_placeholder__::__wbindgen_link_91767d44553c09e6`
    1: wasm-bindgen needs to be invoked with `--split-linked-modules`, because "./not-found.js" cannot be embedded.
       See https://rustwasm.github.io/wasm-bindgen/reference/cli.html#--split-linked-modules for details.
error: test failed, to rerun pass `--test wasm`
```

It is not immediately obvious that the fix is to set environment variable `WASM_BINDGEN_SPLIT_LINKED_MODULES` before running the test suite.

**Comment by flumm** [comment]
should be improved (if not fixed) by #3629

**Comment by daxpedda** [comment]
Thanks @snOm3ad for reporting it and @flumm for fixing it!

Fixed by #3629.
